src/stories/ProfileHeader.stories.tsx

```diff
@@ -23,142 +22,76 @@ type Story = StoryObj<typeof meta>;
-          <span className="profile-header-popup-menu-item-left">
-            <Icon name="user" />
-          </span>
-          <span className="profile-header-popup-menu-item-center">
-            Your Profile
-          </span>
-        </ProfileHeaderPopupMenuItem>
-        <ProfileHeaderPopupMenuItem className="profile-header-popup-menu-item">
-          <span className="profile-header-popup-menu-item-left">
-            <Icon name="notifications" />
-          </span>
-          <span className="profile-header-popup-menu-item-center">
-            Notifications
-          </span>
-          <span className="profile-header-popup-menu-item-right">
-            <svg
-              className="notification-count"
-              xmlns="http://www.w3.org/2000/svg"
-              width="24"
-              height="24"
-              viewBox="0 0 24 24"
-              fill="none">
-              <circle
-                cx="10"
-                cy="12"
-                r="10"
-                fill="#AF4448"
-                style={{
-                  display:
-                    profileHeaderNotifications.length > 0 ? "flex" : "none",
-                }}></circle>
-              <text
-                style={{
-                  display:
-                    profileHeaderNotifications.length > 0 ? "flex" : "none",
-                }}
-                alignmentBaseline="middle"
-                x="10"
-                y="12.5"
-                textAnchor="middle">
-                {profileHeaderNotifications.length}
-              </text>
-            </svg>
-          </span>
-        </ProfileHeaderPopupMenuItem>
-        <ProfileHeaderPopupMenuItem className="profile-header-popup-menu-item">
-          <span className="profile-header-popup-menu-item-left">
-            <Icon name="settings" />
-          </span>
-          <span className="profile-header-popup-menu-item-center">
-            Settings
-          </span>
-        </ProfileHeaderPopupMenuItem>
-        <ProfileHeaderPopupMenuItem className="profile-header-popup-menu-item">
-          <span className="profile-header-popup-menu-item-left">
-            <Icon name="logout" />
-          </span>
-          <span className="profile-header-popup-menu-item-center">Logout</span>
-        </ProfileHeaderPopupMenuItem>
-      </div>
-    </div>
-  );
-};
-
 // More on writing stories with args: https://storybook.js.org/docs/react/writing-stories/args
 export const Example: Story = {
-  render: () => {
-    const [profileHeaderPopupOpen, setProfileHeaderPopupOpen] = useState(false);
-    const [profileHeaderNotifications] = useState([{ type: "alert" }]);
+  render: (args: ProfileHeaderProps) => {
+    const [profileHeaderPopupOpen, setProfileHeaderPopupOpen] = useState(
+      args.profileHeaderPopupOpen
+    );
+
+    // More on writing stories with args: https://storybook.js.org/docs/react/writing-stories/args
+    const menuItems: MenuItemType[] = [
```

Comment:
Let's add `menuItems` and `popupMenuItems` to the story args so they show up in the story form.  Right now there's just an empty object.
<img width="412" alt="image" src="https://github.com/CDCgov/cdc-react/assets/98106992/37bd314f-0d84-46b9-8bf3-b161959383ef">